tensorflow/python/lib/io/file_io.i

```diff
@@ -237,6 +237,67 @@ string ReadFromStream(tensorflow::io::BufferedInputStream* stream,
   return result;
 }
 
+string ReadFromStreamUTF8(tensorflow::io::BufferedInputStream* stream,
+                          size_t bytes,
+                          TF_Status* status) {
+  tensorflow::tstring result;
```

Comment:
Should this be `std::string` or `tensorflow::tstring`? Afaik, `tensorflow:tstring` should be used only if the string is in a tensor. Adding @gharibian to confirm